src/App.jsx

```diff
@@ -57,7 +58,7 @@ function App() {
             <i className="fa-solid fa-gift" />
           </h1>
         </Header>
-        <Overview />
+        <Overview id={id} />
```

Comment:
For consistency, can you access the `id` within the `Overview` component using redux? It can be done with a simple function like `const id = useSelector((state) => state.product.productId)`, just make sure you import the `useSelector` hook.